client/src/components/NotFound/index.tsx

```diff
@@ -1,7 +1,23 @@
+import { Result, Button } from 'antd';
 import * as React from 'react';
+import { useHistory } from 'react-router';
+import { URL_HOME } from '../../constants';
 
 const NotFound: React.FC = () => {
-  return <div>404!</div>;
+  const history = useHistory();
+
+  return (
+    <Result
+      status="404"
+      title="404"
+      subTitle="Sorry, the page you are trying to visit does not exist."
```

Comment:
Seems like we'll have to wait for https://github.com/OpenSRP/web/pull/41 to be merged and update this component accordingly to support multiple languages

Reply:
shouldn't we merge the components first and then go for once over the code and translate it all ?

because if we keep waiting for other branches. it will take our time. what do you say ?

Reply:
@kelvin-muchiri I suggest that we convert this into a package and then merge it after creating an issue to make it translatable